### PR TITLE
SJRK-297: add sojustrepairit theme

### PR DESCRIPTION
**Pull request by BlueSlug**
Adds the new SJRK-flavoured `sojustrepairit` theme, closely modeled after `cities` and `learningReflections`

After merging, a new branch should be created, from which the site will be hosted. Naming could follow the established convention: `stories-sojustrepairit-production`

For more information, including domain, CI info and configuration settings, please see the [SJRK-297 Jira](https://issues.fluidproject.org/browse/SJRK-297)

### CODE REVIEW

**Comment by cindyli** [comment]
It would be helpful to add a comment for [this line](https://github.com/fluid-project/sjrk-story-telling/blob/stories-floe-dev/sjrk.storyTelling.server.config.json5.example#L13) to list all available themes as a reminder, as developers change the theme value from time to time for testing and development. Thanks.

**Comment by BlueSlug** [comment]
I have updated the README and added a couple of notes in the example config file to clarify what options are available. I have also run the browser tests in Chrome 77, Firefox 69 and Edge 44 on Windows 10 v.1903 both from a node server and via the `docker-compose` dev build, and everything passed without issue.

Ready for more review

File: sjrk.storyTelling.server.config.json5.example [hunk 1]
```diff
@@ -6,16 +6,28 @@
...
-                        // a default file to load at the site root (e.g. index.html)
+                        /*
+                         * a default file to load at the site root (e.g. index.html)
+                         * if left blank or not set, this will default to storyBrowse.html
+                         */
                         "themeIndexFile": "introduction.html",
 
-                        // if true, saving stories to the database will be enabled
+                        /*
+                         * if true, saving stories to the database will be enabled
```
Comment (cindyli) [inline sjrk.storyTelling.server.config.json5.example]: It might be more accurate to say "adding and editing stories will be enabled".

Comment (BlueSlug) [reply sjrk.storyTelling.server.config.json5.example]: Good catch! I'll make that change

Comment (BlueSlug) [reply sjrk.storyTelling.server.config.json5.example]: what do you think of "creating and saving stories..." instead? "Editing" might imply that they can be changed after creation (I realize that the page is called "edit")

Comment (cindyli) [reply sjrk.storyTelling.server.config.json5.example]: Ya, the change makes sense.

**Comment by cindyli** [comment]
1. Open any SJRK webpage in a narrower browser;
2. Open UIO and set the text size to 2;
3. Scroll to the the bottom of the page, the section of partner logos continue to display horizontally  and the last IDRC logo is out of the view.

Screen Shot 2019-10-02 at 4 05 26 PM

**Comment by BlueSlug** [comment]
I've adjusted the footer container (three separate rules for three separate themes) to have a maximum width of 100%. The result is that the logos are a little small at the narrowest width before the styling break (50em), but they seem to fit properly at various widths in several browsers.

**Comment by cindyli** [comment]
Merged at a870c8afe9609a484f9005778039a0fe73974545

**Comment by cindyli** [comment]
The new branch for hosting SJRK-flavoured theme is created at https://github.com/fluid-project/sjrk-story-telling/tree/stories-sojustrepairit-production
